inst/scripts/make-grieneisen-ts-metadata.R

```diff
@@ -0,0 +1,70 @@
+
+library(dplyr)
+library(R.utils)
+library(stringr)
+
+BiocVersion <- "3.14"
+path <- paste0("MicrobiomeDataSets/",BiocVersion,"/")

```

Comment:
This should have the package name, it is `microbiomeDataSets` (and not `MicrobiomeDataSets` with uppercase)